tests/conftest.py

```diff
@@ -20,7 +20,7 @@
 from uuid import uuid4
 
 import pytest
-from py._builtin import execfile
+from pytest import File
```

Comment:
Why this change is needed? Conftest is used across all test suites and we need to justify such a change somehow.

Reply:
In the PR description it mentions that `execfile` is deprecated, but agreed it's a sensitive change.

Reply:
I see, well let's give CI a shot then and see if something is affected

Reply:
The `py._builtin` module was part of older versions of `pytest `and is now deprecated.